### DISCUSSION TITLE
Validação de formulário

### DISCUSSION THREAD

**Discussion by danieloprado** [eduzz/front-end#2] [Ideas]
Como fazer a validação? react-form-hooks + yup / Antd Form

**Comment by ffernandomoraes** [comment]
Eu gosto de como o Antd lida com os formulários, eu acho muito bom e simples (nunca tive problema). As vezes pode até ser um pouco repetitivo a parte de rules dele (não sei se tem algo que podemos fazer pra melhorar isso), porém não vejo problemas nisso, tendo em vista que a maioria dos formulários da Eduzz é bem simples.

Mas o principal ponto que eu vejo problema em não adotar o padrão do Antd, é que vamos precisar ficar adaptando e mantendo libs para contornar isso. Que foi o caso do @eduzz/houston-forms. Sem contar as documentações que precisamos ficar gerando e as dúvidas dos times que surgem durante o processo.

**Reply by miguelaugl** [reply]
Concordo!
